### PR TITLE
feat: add terraform factory and yaml factory generator

### PR DESCRIPTION
**Pull request by edyw**

### CODE REVIEW

File: tools/custom-organization-policy-library/Makefile [hunk 1]
```diff
@@ -2,21 +2,34 @@
 SHELL 	:= /usr/bin/env bash
 CONFIG 	:= -f build/config/schema.yaml -f build/config/services/ -f values.yaml -f build/ytt_lib/
 OUTPUT 	:= samples
+OUTPUT_TF := tf/data
```
Comment (vannicktrinquier) [inline tools/custom-organization-policy-library/Makefile:5]: updates samples folder to have both like samples/gcloud and samples/terraform

File: tools/custom-organization-policy-library/Makefile [hunk 2]
```diff
@@ -2,21 +2,34 @@
...
 
 .PHONY: constraints
 constraints: 
 	rm -rf $(OUTPUT)/constraints
 	ytt $(CONFIG) -f build/custom-constraints/  --output-files $(OUTPUT)/constraints
 
+.PHONY: constraints-tf
+constraints-tf: constraints
+	python3 scripts/cvt-tf-constraints.py $(OUTPUT)/constraints tf/data/custom-constraints
```
Comment (vannicktrinquier) [inline tools/custom-organization-policy-library/Makefile:14]: path to be update to go to good output path

File: tools/custom-organization-policy-library/tf/data/custom-policies/custom.computeAllowedDiskTypes.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+custom.computeAllowedDiskTypes:
```
Comment (vannicktrinquier) [inline tools/custom-organization-policy-library/tf/data/custom-policies/custom.computeAllowedDiskTypes.yaml]: This format is compatible with FAST ?

Comment (edyw) [reply tools/custom-organization-policy-library/tf/data/custom-policies/custom.computeAllowedDiskTypes.yaml]: yes

**Pull request closed without merging**